packages/api/src/controllers/taskController.js

```diff
@@ -83,37 +84,54 @@ const taskController = {
           abandon_date,
         } = req.body;
 
-        const { owner_user_id, assignee_user_id, wage_at_moment, override_hourly_wage } = await TaskModel.query()
+        const {
+          owner_user_id,
+          assignee_user_id,
+          wage_at_moment,
+          override_hourly_wage,
+        } = await TaskModel.query()
           .select('owner_user_id', 'assignee_user_id', 'wage_at_moment', 'override_hourly_wage')
-          .where({ task_id }).first();
+          .where({ task_id })
+          .first();
         const isUserTaskOwner = user_id === owner_user_id;
         const isUserTaskAssignee = user_id === assignee_user_id;
         const hasAssignee = assignee_user_id !== null;
         // TODO: move to middleware
         // cannot abandon task if user is worker and not assignee and not creator
         if (!adminRoles.includes(req.role) && !isUserTaskOwner && !isUserTaskAssignee) {
-          return res.status(403).send('A worker who is not assignee or owner of task cannot abandon it');
+          return res
+            .status(403)
+            .send('A worker who is not assignee or owner of task cannot abandon it');
         }
         // cannot abandon an unassigned task with rating or duration
         if (!hasAssignee && (happiness || duration)) {
-          return res.status(406).send('An unassigned task should not be rated or have time clocked');
+          return res
+            .status(406)
```

Comment:
406 is about content negotiation, so it doesn't seem appropriate here. Maybe 409, or just generic 400.